newIDE/app/src/EventsSheet/InlinePopover.js

```diff
@@ -54,10 +55,16 @@ export default function InlinePopover(props: Props) {
   return (
     <ClickAwayListener
       onClickAway={event => {
+        // Clicks on dialogs (or their backdrop element) generated by
+        // the AlertProvider seem to trigger this click away listener
+        // whereas they are displayed above the InlinePopover element.
```

Comment:
replace whereas => when

Reply:
I mean whereas so... 😅 . The inline popover is only used in the event sheet, dialogs will always be displayed above it